net/sched/sch_skbprio.c

```diff
@@ -239,6 +223,7 @@ static int skbprio_dump(struct Qdisc *sch, struct sk_buff *skb)
 static void skbprio_reset(struct Qdisc *sch)
 {
 	struct skbprio_sched_data *q = qdisc_priv(sch);
+	const unsigned int min_limit = 1;
```

Comment:
Remember to drop this line once the code below that is using it goes away (see another comment).